save-plugins/fix-and-warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fixandwarn/FixAndWarnPluginConfig.kt

```diff
@@ -26,14 +26,19 @@ data class FixAndWarnPluginConfig(
     override var configLocation: Path = "undefined_toml_location".toPath()
     override val resourceNamePatternStr: String = "(${fix.resourceNamePatternStr})|(${warn.resourceNamePatternStr})"
 
+    @Transient
```

Comment:
why `@Transient`? I though that we deserialize this value from toml config

Reply:
The point it that we deserialize `ignoreLines` from toml config. `ignoreLinesPatterns` is initialized with `ignoreLines` that we parse from config so it is not deserialized directly from toml.